build.sbt

```diff
@@ -10,7 +10,8 @@ libraryDependencies ++= Seq(
   "org.apache.daffodil" %% "daffodil-tdml-processor" % "3.4.0" % "test",
```

Comment:
Should this become version 1.3.1 (above. Github won't let me put a comment directly on that line.)